app/counters/workflow_counter.rb

```diff
@@ -7,10 +7,6 @@ def initialize(workflow)
   end
 
   def classifications
-    scope = workflow.classifications
-    if launch_date = workflow.project.launch_date
-      scope = workflow.classifications.where("created_at >= ?", launch_date)
-    end
-    scope.count
+    SubjectWorkflowStatus.where(workflow: workflow).sum(:classifications_count)
```

Comment:
Yeah - i tend to agree about the numbers not validating with the current linked set. I'll iterate on this in another PR.